agent/containers/images/Makefile

```diff
@@ -32,108 +38,115 @@ _WORKLOAD_RPMS = fio uperf
 # Not intended to be overridden with an environment variable.
 _ALL_RPMS = ${_TOOL_RPMS} ${_WORKLOAD_RPMS}
 
+_DISTROS = centos-8 centos-7 fedora-32 fedora-31
+
 # Default we only build images the following distributions:
```

Comment:
This is both ungrammatical and misplaced: it should be above l.41.

Reply:
Fixed.